app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (VaccineError, NotWearingMaskError)
```

Comment:
Remove redundant brackets
```suggestion
from app.errors import VaccineError, NotWearingMaskError
```